packages/jest-matchers/src/__tests__/matchers-test.js

```diff
@@ -55,6 +55,21 @@ describe('.toBe()', () => {
     expect(() => jestExpect(obj).toBe({}))
       .toThrowErrorMatchingSnapshot();
   });
+
+  test(`assertion error matcherResult property contains 
+  matcher name, expected and actual values`, () => {
```

Comment:
can you keep this in one line? I fixed the lint rule in https://github.com/facebook/jest/pull/2575